### PR TITLE
Add vote points to set up baseUri for withdrawal nft

### PR DESCRIPTION
**Pull request by folkyatina**

### CODE REVIEW

File: scripts/upgrade_shapella.py [hunk 1]
```diff
@@ -30,14 +30,18 @@
 25. Revoke `SET_BEACON_SPEC` role from `Voting`
 26. Revoke `SET_REPORT_BOUNDARIES` role from `Voting`
 27. Revoke `SET_BEACON_REPORT_RECEIVER` role from `Voting`
+28. Grant a `MANAGE_TOKEN_URI_ROLE` role to `Voting`
```
Comment (TheDZhon) [inline scripts/upgrade_shapella.py:33]: Grant *the* `MANAGE_TOKEN_URI_ROLE` role :) or exclude the article as above

File: scripts/upgrade_shapella.py [hunk 2]
```diff
@@ -202,6 +225,9 @@ def start_vote(tx_params: Dict[str, str], silent: bool) -> Tuple[int, Optional[T
         "25) Revoke `SET_BEACON_SPEC` role from `Voting`",
         "26) Revoke `SET_REPORT_BOUNDARIES` role from `Voting`",
         "27) Revoke `SET_BEACON_REPORT_RECEIVER` role from `Voting`",
+        "28) Assign a `MANAGE_TOKEN_URI_ROLE` role to Voting",
```
Comment (TheDZhon) [inline scripts/upgrade_shapella.py:228]: lost slashing quotes for `Voting`, extra 'a' article

File: tests/test_upgrade_shapella.py [hunk 3]
```diff
@@ -328,9 +331,18 @@ def test_vote(
         events_update_oracle_impl,
         events_grant_staking_router_role,
         events_template_finish,
-        *revoke_roles_events,
+        *remaining_events,
```
Comment (TheDZhon) [inline tests/test_upgrade_shapella.py:334]: maybe it's better to have two groups of events to avoid magic-border slices below?

**Review by TheDZhon — APPROVED** [review]
Thank you! :+1: 

Please take a look at a few minor suggestions at your convenience.

File: tests/test_upgrade_shapella.py (hunk 3 above)
Comment (folkyatina) [reply tests/test_upgrade_shapella.py:334]: How do you see it? I don't catch.
And "magical" slices is not scary at all here. It's properly checked afterwards that we sliced exactly what is meant  to be sliced.

**Review by bulbozaur — APPROVED** [review]

File: tests/test_upgrade_shapella.py (hunk 3 above)
Comment (TheDZhon) [reply tests/test_upgrade_shapella.py:334]: It's not so important to me, so I took your point :+1:

File: scripts/upgrade_shapella.py [hunk 4]
```diff
@@ -30,14 +30,18 @@
 25. Revoke `SET_BEACON_SPEC` role from `Voting`
 26. Revoke `SET_REPORT_BOUNDARIES` role from `Voting`
 27. Revoke `SET_BEACON_REPORT_RECEIVER` role from `Voting`
+28. Grant `MANAGE_TOKEN_URI_ROLE` role to `Voting`
+29. Set `WithdrawalQueueERC721` baseUri to `https://wq-api.lido.fi/v1/nft`
+30. Revoke `MANAGE_TOKEN_URI_ROLE` role from `Voting`
```
Comment (kadmil) [inline scripts/upgrade_shapella.py:35]: Do we need to revoke though?

Comment (TheDZhon) [reply scripts/upgrade_shapella.py:35]: Better to revoke. Will be sanitizing roles for the Dual Gov anyways.

Comment (folkyatina) [reply scripts/upgrade_shapella.py:35]: Actually, there is another way. 
We can grant the role to Agent, not Voting and leave it there. But, according to Occam razor, you don't want to multiply entities, so I'd rather clean it up.

**Pull request merged**